provision/ubuntu/kernel-next.sh

```diff
@@ -27,6 +27,9 @@ rm -rf pahole
 
 export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib/
 
+# Apply local patches
+git apply < /tmp/0001-cpumask-Omit-terminating-null-byte-in-cpumap_print_-.patch
```

Comment:
Making it more generic would just be a matter of replacing here with the following, no?
```suggestion
git apply < /tmp/*.patch
```

Reply:
Done